<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamplesWindow</class>
 <widget class="QMainWindow" name="ExamplesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Startup window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="cbDoNotShowExamplesOnStart">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>433</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Do not show this window on startup</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>289</y>
      <width>457</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Configuration file</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>55</y>
      <width>451</width>
      <height>233</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>114</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Description of the example:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLoadExample">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>204</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Load this example</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>4</y>
       <width>107</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Select type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="lwMainType">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>20</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QTextEdit" name="teExampleDescription">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>131</y>
       <width>441</width>
       <height>70</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="lwSubType">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>20</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>4</y>
       <width>178</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Select example file:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>4</y>
       <width>130</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Select branch:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="lwExample">
     <property name="geometry">
      <rect>
       <x>266</x>
       <y>20</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>36</y>
      <width>457</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Examples</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>308</y>
      <width>451</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="cbSkipDetectorConstructor">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>21</y>
       <width>150</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Detector constructor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSkipSimConfig">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>38</y>
       <width>150</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Simulation settings</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbSkipRecConfig">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>56</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Reconstruction settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLoadSettings">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbQuickLoad">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>87</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Quick load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbQuickSave">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>87</y>
       <width>111</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Quick save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSaveSessings">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>71</y>
       <width>430</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>3</y>
       <width>173</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip the following (if applicable):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>90</y>
       <width>172</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Load/Save using a fixed slot</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>8</y>
       <width>3</width>
       <height>62</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>8</y>
       <width>3</width>
       <height>62</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>88</y>
       <width>3</width>
       <height>24</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>276</x>
       <y>88</y>
       <width>3</width>
       <height>24</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbNew">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>6</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Create new detector</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>29</y>
      <width>460</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
